main.rb

```diff
@@ -65,6 +66,30 @@ def options(input)
     end
   end
 
+  def add_game
+    multiplayer = user_input("Multiplayer: ")
+    last_played = user_input('Last played at: ')
+    publish_date = user_input('Publish date: ')
+    Game.new(multiplayer, last_played, publish_date).add_game
+    puts "The Game with #{multiplayer} as mulptiplayer has been created successfully ✅"
+  end
+
+  def list_games
+    File.new('games.json', 'w+') unless Dir.glob('*.json').include? 'games.json'
+
+    if File.empty?('games.json')
+      games = []
+    else
+      data = File.read('games.json').split
+      games = JSON.parse(data.join)
+    end
+     puts 'List of Games'
+    games.each_with_index do |game, key|
+      puts "#{key + 1}) Multiplayer: #{game['multiplayer']} Last Played: #{game['last_played_at']} Publish Date: #{game['publish_date']}"
+    end
```

Comment:
It's better practice if you put a message that says something like "List is empty" when there are no games stored yet. Please consider adding it in your code.